wscript

```diff
@@ -649,6 +649,11 @@ video_output_features = [
         'desc': 'VAAPI (Wayland support)',
         'deps': 'vaapi && gl-wayland',
         'func': check_pkg_config('libva-wayland', '>= 1.1.0'),
+    }, {
+        'name': 'vaapi-wayland-memfd',
+        'desc': 'VAAPI (Wayland support)',
+        'deps': 'vaapi',
+        'func': check_pkg_config('libva-wayland', '>= 1.1.0'),
```

Comment:
This one can just be `deps: vaapi-wayland && memfd_create` and `func: check_true`.